source/guides/exchanges/exchange-integration.rst

```diff
@@ -0,0 +1,353 @@
+############################
+Integrating with an Exchange
+############################
+
+This document is intended to **guide developers** through the integration of the ``XYM`` token into an **Exchange platform**. It contains recommendations on
+how to set up accounts, listen for deposits, and create withdrawals as well as code examples ready to be adopted.
+
+The code examples shared use the `Symbol SDK for TypeScript <https://github.com/nemtech/symbol-sdk-typescript-javascript>`__, but can be ported to other :doc:`available SDKs <../../sdk>` since all of them share the same design principles. If there is no SDK supported for a required
+programming language, you may still be able to integrate by connecting directly via Symbol's :doc:`REST API <../../api>`.
+
+Integration overview
+********************
+
+There are many ways to design an exchange. This guide is based on how to support ``XYM`` deposits and withdrawals in an exchange that follows a **central wallet approach**.
+
+Please note that this design is not particularly recommend over others. However, its **simplified architecture** is a good showcase for Symbol's set of features involved in integrating with an Exchange. A different approach, for example, would be to use a different wallet for each user.
+
+.. figure:: /resources/images/diagrams/exchange-integration-overview.png
+   :align: center
+   :width: 100%
+   :target: /_images/exchange-integration-overview.png
+
+   **Fig. 1**: General design diagram of the central wallet approach.
+
+The main components of this architecture are described next.
+
+Components
+==========
+
+Central wallet
```

Comment:
I like to emphasize that it is "Central", because it is in the middle of everything (all money goes through it). But I'll add a note calling it also "Hot", because that is probably a more popular term.